src/ae.c

```diff
@@ -103,7 +103,13 @@ int aeGetSetSize(aeEventLoop *eventLoop) {
     return eventLoop->setsize;
 }
 
-/* Tells the next iteration/s of the event processing to set timeout of 0. */
+/*
+ * Tell the event processing to change the wait timeout as soon as possible.
+ *
+ * Note: that this does not always take effect. Specifically, the parameter flags
+ * be set to AE_DONT_WAIT when calling aeProcessEvent(), then this operation will
+ * be ignored.
+ */
```

Comment:
i don't think that long comment makes sense, if you pass `0` to disable the `noWait`, it doesn't mean you instruct the event loop to wait (and as you say that can be ignored).
it just means you turn off the global DONT_WAIT.

Reply:
Yes, the reason why I modified this comment is that this function is somewhat confusing in my opinion.
However, your statement seems more explicit. 
So, how about I change the ```Note:``` to ```it just means you turn off the global DONT_WAIT```?

Reply:
The comment is updated.